### DISCUSSION TITLE
✅ Partial Sliders - Loop Grid & Carousels

### DISCUSSION THREAD

**Discussion by leeKIJO** [elementor/elementor#24872] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

Add the options for partial sliders. Whether that be 2.5 or px or %

### Describe the solution you'd like.

Add the options for partial sliders. Whether that be 2.5 or px or %

### Additional context

While using the loop grid and loop carousel you can choose how many slides etc you would like to show at each breakpoint. Add the options for partial sliders. Whether that be 2.5 slides or 2 slides and  60px etc of the 3rd slide or %

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by corgalore** [comment]
I agree. The Swiper JS already supports using decimal slide counts; please change the Slides on Display control from a Select to a Number so we can enter any value we need. That way we no longer need to enable Infinite Scroll in order to get the offset behavior.

**Comment by nicholaszein** [comment]
Hi, this feature was added to the Nested Carousel widget (Pro), and Loop Carousel widget (Pro) in version 3.15.0. It's called "Offset Sides".

You can learn more about it in the links below: 
- [Release Post](https://elementor.com/blog/introducing-315-taxonomy-filter-nested-accordion-and-more/#:~:text=%5BPro%5D%20Enhanced%20Styling%20Capabilities%20in%20the%20Carousel%20Widgets)
- [WebSquadron video tutorial](https://youtu.be/Kx_ew2HNz1c?si=QgKPTzeMs7eJnTbF)

I'm closing this request as solved.

Kind regards
